src/include/c_api/kuzu.h

```diff
@@ -0,0 +1,253 @@
+KUZU_C_API kuzu_internal_id_t kuzu_value_get_internal_id(kuzu_value* value);
+KUZU_C_API kuzu_node_val* kuzu_value_get_node_val(kuzu_value* value);
+KUZU_C_API kuzu_rel_val* kuzu_value_get_rel_val(kuzu_value* value);
+KUZU_C_API kuzu_date_t kuzu_value_get_date(kuzu_value* value);
+KUZU_C_API kuzu_timestamp_t kuzu_value_get_timestamp(kuzu_value* value);
+KUZU_C_API kuzu_interval_t kuzu_value_get_interval(kuzu_value* value);
+KUZU_C_API char* kuzu_value_get_string(kuzu_value* value);
+KUZU_C_API char* kuzu_value_to_string(kuzu_value* value);
+
+KUZU_C_API kuzu_node_val* kuzu_node_val_create(kuzu_internal_id_t id, char* label);
+KUZU_C_API kuzu_node_val* kuzu_node_val_clone(kuzu_node_val* node_val);
+KUZU_C_API void kuzu_node_val_destroy(kuzu_node_val* node_val);
+KUZU_C_API kuzu_value* kuzu_node_val_get_id_val(kuzu_node_val* node_val);
+KUZU_C_API kuzu_value* kuzu_node_val_get_label_val(kuzu_node_val* node_val);
+KUZU_C_API kuzu_internal_id_t kuzu_node_val_get_id(kuzu_node_val* node_val);
+KUZU_C_API char* kuzu_node_val_get_label_name(kuzu_node_val* node_val);
+KUZU_C_API uint64_t kuzu_node_val_get_property_size(kuzu_node_val* node_val);
+KUZU_C_API char* kuzu_node_val_get_property_name_at(kuzu_node_val* node_val, uint64_t index);
+KUZU_C_API kuzu_value* kuzu_node_val_get_property_value_at(kuzu_node_val* node_val, uint64_t index);
+KUZU_C_API void kuzu_node_val_add_property(
+    kuzu_node_val* node_val, const char* key, kuzu_value* value);
+KUZU_C_API char* kuzu_node_val_to_string(kuzu_node_val* node_val);
+
+KUZU_C_API kuzu_rel_val* kuzu_rel_val_create(
+    kuzu_internal_id_t src_id, kuzu_internal_id_t dst_id, char* label);
+KUZU_C_API kuzu_rel_val* kuzu_rel_val_clone(kuzu_rel_val* rel_val);
+KUZU_C_API void kuzu_rel_val_destroy(kuzu_rel_val* rel_val);
+KUZU_C_API kuzu_value* kuzu_rel_val_get_src_id_val(kuzu_rel_val* rel_val);
+KUZU_C_API kuzu_value* kuzu_rel_val_get_dst_id_val(kuzu_rel_val* rel_val);
+KUZU_C_API kuzu_internal_id_t kuzu_rel_val_get_src_id(kuzu_rel_val* rel_val);
+KUZU_C_API kuzu_internal_id_t kuzu_rel_val_get_dst_id(kuzu_rel_val* rel_val);
+KUZU_C_API char* kuzu_rel_val_get_label_name(kuzu_rel_val* rel_val);
+KUZU_C_API uint64_t kuzu_rel_val_get_property_size(kuzu_rel_val* rel_val);
+KUZU_C_API char* kuzu_rel_val_get_property_name_at(kuzu_rel_val* rel_val, uint64_t index);
+KUZU_C_API kuzu_value* kuzu_rel_val_get_property_value_at(kuzu_rel_val* rel_val, uint64_t index);
+KUZU_C_API void kuzu_rel_val_add_property(kuzu_rel_val* rel_val, char* key, kuzu_value* value);
+KUZU_C_API char* kuzu_rel_val_to_string(kuzu_rel_val* rel_val);
+
+// QuerySummary
+KUZU_C_API void kuzu_query_summary_destroy(kuzu_query_summary* query_summary);
+KUZU_C_API double kuzu_query_summary_get_compiling_time(kuzu_query_summary* query_summary);
+KUZU_C_API double kuzu_query_summary_get_execution_time(kuzu_query_summary* query_summary);
+
+// TODO: Bind utility functions for kuzu_date_t, kuzu_timestamp_t, and kuzu_interval_t
```

Comment:
Can you open an issue on this? We can address this later.

Reply:
Opened #1476